worker/build.rs

```diff
@@ -93,20 +92,6 @@ fn main() {
         {
             panic!("Failed to build libmediasoup-worker")
         }
-
-        if env::var("KEEP_BUILD_ARTIFACTS") != Ok("1".to_string()) {
-            // Clean
-            if !Command::new("make")
-                .arg("clean-all")
-                .spawn()
-                .expect("Failed to start")
-                .wait()
-                .expect("Wasn't running")
-                .success()
-            {
-                panic!("Failed to clean libmediasoup-worker")
-            }
-        }
```

Comment:
This is important for package publishing for instance. But yes, `MEDIASOUP_OUT_DIR` is missing here and needs to be added.